kyaml/fn/framework/gosdk/command/command.go

```diff
@@ -0,0 +1,54 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/gosdk"
+)
+
+type Options struct {
+	NoPrintError bool
+}
+
+// AsMain should be called from main.
+func AsMain(ctx context.Context, fn gosdk.KRMFunction, options Options) *cobra.Command {
```

Comment:
I am not sure why we use cobra for functions.  Aren't functions just called with a bunch of YAML and not provided parameters?

Reply:
Updated